dashboards/argo-workflows/argo-workflows-prometheus.json

```diff
@@ -17,115 +17,125 @@
       "templateVariable": "Namespace"
     }
   ],
-  "displayName": "Argo Workflows Prometheus Overview",
-  "labels": {},
   "mosaicLayout": {
-    "columns": 12,
+    "columns": 48,
     "tiles": [
       {
-        "height": 2,
+        "xPos": 8,
+        "width": 8,
+        "height": 8,
         "widget": {
+          "title": "Pending",
           "scorecard": {
             "blankView": {},
+            "thresholds": [],
             "timeSeriesQuery": {
-              "prometheusQuery": "argo_workflows_count{status=\"Pending\",${Cluster},${Location},${Namespace}}"
+              "prometheusQuery": "argo_workflows_count{status=\"Pending\",${Cluster},${Location},${Namespace}}",
```

Comment:
can all of these scorecards be wrapped in `sum(...)`?

I think there could hypothetically be pending/running/etc stuff on multiple pods, and we'd want the total across pods.